### DISCUSSION TITLE
How to use full width

### DISCUSSION THREAD

**Discussion by BenVdP** [microsoft-search/pnp-modern-search#3062] [Q&A]
Hi,

I want to know if it is possible to make the search results webpart use the full width of a page section.
As you can see below on a small(er) screen it looks like it is using full width, but the columns are not readable, if I change de column widths, it adds an ugly scrollbar...

![image](https://github.com/microsoft-search/pnp-modern-search/assets/57953469/7f4bdaa2-d8ae-42fa-842c-61f7941e973a)

On a big(ger) screen there is a lot of white space to right of the results webpart, eventhough if I change the column widths, it is still adding this scrollbar.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/57953469/04fc4267-0720-4c12-9cbf-e45ee9486512)

Is there something I can do to present the results in a user friendly way?

**Comment by kasperbolarsen** [comment]
Excellent question 👍

**Comment by wobba** [comment]
Can you propose how you want the web part to space columns when there is more information than space? And we cannot use more space than the page section allows. In your screenshot the 1/3 left column has space, and then the 2/3 column is also restricted in max width by SharePoint so we cannot make it flow beyond that.

I'm sure you hack some CSS to make it go beyond, but that would be a hack.

**Comment by BenVdP** [comment]
@wobba , I would like to space the columns as the webpart is now in a 'one column' section. 
![2023-06-13_11-41-48](https://github.com/microsoft-search/pnp-modern-search/assets/57953469/1cc86f4f-b395-4153-98d9-ec728c4098fc)

I didn't want to move the filters to another location as they are now less visible on the page, but I had to to make more room for the results webpart to render all the columns properly.

Hope this makes sense?

**Reply by wobba** [reply]
Not really. When places in a region/section like your first screenshot. How should it appear different from what you shown? It's using all available space.
